examples/v1beta1/taskruns/optional-resources-with-clustertask.yaml

```diff
@@ -1,36 +0,0 @@
-# Note: ClusterTasks are deprecated. Please use the cluster resolver instead.
-# https://github.com/tektoncd/pipeline/blob/main/docs/cluster-resolver.md
-apiVersion: tekton.dev/v1beta1
-kind: ClusterTask
-metadata:
-  name: clustertask-with-optional-resources-v1beta1
-spec:
-  params:
-  - name: filename
-    type: string
-    default: "README.md"
-  resources:
-    inputs:
-      - name: git-repo
-        type: git
-        optional: true
-    outputs:
```

Comment:
I'd prefer just removing image resources from the examples that don't rely on building images rather than removing the examples entirely.

Reply:
Thanks I will restore them.
Previously I saw them as cases that would only be meaningful if the outputs resources could be used as a digest. But that makes sense to leave the rest of git resources until removal of all resources.